.github/workflows/test.yml

```
@@ -32,10 +29,13 @@ jobs:
         run: |
-          rustup toolchain install ${{ matrix.toolchain }} --profile minimal --no-self-update
-          rustup default ${{ matrix.toolchain }}
-          rustup component add rustfmt --toolchain ${{ matrix.toolchain }}
-          rustup component add clippy --toolchain ${{ matrix.toolchain }}
+          rustup toolchain install --profile minimal --no-self-update
+          rustup component add rustfmt
+          rustup component add clippy
+
+      - name: Cargo Tools
+        uses: taiki-e/install-action@v2
+        tool: cargo-binstall
```

actionlint (1.7.8)

unexpected key "tool" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)